polygeist/tools/cgeist/Test/Verification/builtin.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: cgeist -O0 %s -w -o - -S --function=* | FileCheck %s
+
+// bswap
```

Comment:
NIT: We usually prepend comments with `// COM:` in test files lately.

Reply:
fixed